aries-site/src/pages/components/tabs.mdx

```diff
@@ -42,6 +43,25 @@ To maintain a clean, consistent presentation, all content should be aligned to t
 - Apply `justify="start"` to left-align the tabs and have their bottom border extend the full-width of the parent container.
 - Content within the Tab should be flush with the left edge of the Tabs component and have a vertical padding of `medium` to provide spacing and room for the content to breathe.
 
+### Responsive Tabs
+
+When horizontal space is limited, only the tabs that fit will be displayed. The remaining tabs can be accessed by using the previous and next arrows or with the keyboard.
+
+### Responsive Behavior
```

Comment:
Seems like this is a child of Responsive Tabs.

```suggestion
#### Responsive Behavior
```

Reply:
Fixed in latest commit